### PR TITLE
PFM-2843--Organization module changes

### PR DESCRIPTION
**Pull request by Vasanth26-egov**

### CODE REVIEW

File: data/state1/works/OrganisationClassification.json
Comment (jagankumar-egov) [inline data/state1/works/OrganisationClassification.json:1]: Not required

File: data/state1/works/OrganisationType.json
Comment (jagankumar-egov) [inline data/state1/works/OrganisationType.json:1]: Not required in UI

**Review by jagankumar-egov — CHANGES_REQUESTED** [review]

File: data/state1/works/OrganisationClassification.json
Comment (Vasanth26-egov) [reply data/state1/works/OrganisationClassification.json:1]: Deleted

File: data/state1/works/OrganisationType.json
Comment (Vasanth26-egov) [reply data/state1/works/OrganisationType.json:1]: Deleted

**Review by jagankumar-egov — APPROVED** [review]

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**